samples/cellular/modem_shell/src/sock/sock_shell.c

```diff
@@ -32,7 +32,14 @@ enum sock_shell_command {
 	SOCK_CMD_GETADDRINFO,
 	SOCK_CMD_SEND,
 	SOCK_CMD_RECV,
-	SOCK_CMD_RAI
+	SOCK_CMD_RAI,
+	SOCK_CMD_OPTION_SET,
+	SOCK_CMD_OPTION_GET,
+};
+
+enum option_shell_command {
+	OPTION_CMD_SET,
+	OPTION_CMD_GET,
```

Comment:
To make this clear, I'd use socktopt in one way or the other in the enum name.
```suggestion
enum sockopt_cmd_type {
	SOCKOPT_CMD_TYPE_SET,
	SOCKOPT_CMD_TYPE_GET,
```